utils/validators/formRules/index.js

```diff
@@ -0,0 +1,383 @@
+import { RBAC } from '@/config/types';
+import { HCI } from '@/config/labels-annotations';
+import isEmpty from 'lodash/isEmpty';
+import has from 'lodash/has';
+// import uniq from 'lodash/uniq';
+import cronstrue from 'cronstrue';
+
+const httpsKeys = [
+  'server-url'
+];
+
+const runValidators = (val, validators) => {
+  for (const validator of validators) {
+    const message = validator(val);
+
+    if (message) {
+      return message;
+    }
+  }
+};
+
+// "t" is the function name we use for getting a translated string
+export default function(t, opt = {}) {
+  const { displayKey = 'Value' } = opt;
```

Comment:
@Sean-McQ How would this get localised? We talked about being able to pass in the displayKey - that would allow us to use a default from the control itself, which could use the label for this.

Reply:
So this latest version will replace "Value" in the error message string with the label at the field level using the label (assuming the field has a label. That being said, we could also create a new translation string to use if there's no displayKey.